### DISCUSSION TITLE
Homebrew Install vs Python Install

### DISCUSSION THREAD

**Discussion by bwrichards-ptrn** [AltimateAI/vscode-dbt-power-user#132] [Q&A] [answered]
Quick question!

I have dbt installed on my machine and can verify that dbt is installed by running dbt --version in my terminal. Despite this, dbt Power User tells me that dbt is not installed when attempting to use the extension. 

I installed DBT via Homebrew. Does DBT need to be installed via Python in order for the extension to work?

**Comment by mdesmet** [comment, marked as answer]
You just need to point to the python environment where dbt is installed.

Check out https://github.com/innoverio/vscode-dbt-power-user/issues/87
